cmd/pathlist.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import 	(
+	"fmt"
+	"os"
+	"regexp"
+	"bufio"
+	"io"
+	"sort"
+
+	"github.com/spf13/cobra"
+)
+
+const (
+	regex = "data/[A-Za-z0-9_./]+\\.[a-zA-Z0-9]+"
+)
+
+var pattern regexp.Regexp
+
+func init() {
+	pathlistCmd.AddCommand(pathlistScanCmd())
+	pattern = *regexp.MustCompile(regex)
+}
+
+var pathlistCmd = &cobra.Command{
+	Use:   "pathlist",
+	Short: "Work with pathlist files",
+}
+
+func pathlistScanCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "scan",
+		Short: "Scan a core file for interesting paths",
+		Args: cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			verbose, _ := cmd.PersistentFlags().GetBool("verbose")
+			reader, openErr := os.Open(args[0])
+			r := bufio.NewReader(reader)
+			if openErr != nil {
+				fmt.Fprintln(os.Stderr, openErr)
+				os.Exit(1)
+			}
```

Comment:
Should add `defer reader.Close()` after the error check to ensure the file handle is properly closed

Reply:
:boom: